package day44_Exceptions;

public class CheckedException {

    public static void main(String[] args) {

        System.out.println(" Test started");

       //Thread.sleep(3000); you have to handle right away first

        System.out.println("Done");

    }
}
